lib/vector/Vlib/build_nat.c

```diff
@@ -116,7 +116,7 @@ int Vect_build_nat(struct Map_info *Map, int build)
 	G_progress(1, 1);
 
 	G_verbose_message(n_("One primitive registered", "%d primitives registered", plus->n_lines), plus->n_lines);
-	G_verbose_message(n_("One vertex registered", "%jd vertices registered", npoints), npoints);
+	G_verbose_message(n_("One vertex registered", "%jd vertices registered", npoints), (intmax_t)npoints);
```

Comment:
Automatically casting an integer to a larger type is safe, the explicit cast to intmax_t should not be needed.
Does intmax_t exist on all supported platforms?

Reply:
Changed format specifier, now using PRId64.